pages/instructorAssessmentManualGrading/instanceQuestion/gradingPanel.ejs

```diff
@@ -0,0 +1,82 @@
+<form name="manual-grading-form" method="POST"
+      data-max-auto-points="<%= assessment_question.max_auto_points %>"
+      data-max-manual-points="<%= assessment_question.max_manual_points %>"
+      data-max-points="<%= assessment_question.max_points %>">
+  <input type="hidden" name="__csrf_token" value="<%= __csrf_token %>">
+  <input type="hidden" name="modified_at" value="<%= instance_question.modified_at %>">
+  <div class="form-group row justify-content-center">
+    <label class="custom-control-inline col-auto mx-0">
+      <span class="">Points</span>
+      <div class="custom-control custom-switch mx-2">
+        <input class="custom-control-input js-manual-grading-pts-perc-select"
+               name="use_score_perc" type="checkbox">
+        <span class="custom-control-label">Percentage</span>
+      </div>
+    </label>
+  </div>
+  <%- include('gradingPointsSection',
+    { type: 'manual', type_label: 'Manual',
+      points: locals.manual_points ?? instance_question.manual_points,
```

Comment:
> I had a question that was purely auto-graded. I added a rubric for it with Maximum Points: 4. But in the UI it still displayed the max points as being 0.

These are different values with different fields, but I can see where you get the confusion. The rubric's maximum points is the ceiling for points computed from rubric items. It is different from the number of points the question is worth. So you can have a question that is worth 3 points, and based on the rubrics you can have them go at most 3 points, or at most e.g., 4 points (if you want to allow a rubric for bonus), or at most 2 points (unlikely to be used, but would force an adjustment to reach the maximum). This is the opposite of the floor (minimum points in the settings), which keeps the question from going below 0 for penalties, though you can have that value be -1 (if you allow some penalty that affects e.g., autopoints) or 1 (again unlikely, but that ensures the question will always have at least one point).

What is shown in the grading panel is how many manual points are assigned to the question, which indeed is zero. Since the question's points are based on auto-points only, this would make sense.

Do you have a suggestion for a better way to name these to avoid this confusion?

Reply:
Is this clearer?

<img width="416" alt="image" src="https://user-images.githubusercontent.com/1004907/232312965-41dd414d-2b85-4848-8fb4-c4b8c6ef0089.png">

Reply:
Changed to "max extra points", so this confusion shouldn't come up.